library/src/math/class.rs

```diff
@@ -0,0 +1,155 @@
+use super::*;
+
+/// Math operator class.
+/// Note this doesn't scale the character nor upright the text.
+/// Default contains large operators such as `sum`, `product`.
+///
+/// ## Example
+/// ```example
+/// $ a operator(x o r) b, a x o r b $
+/// ```
+///
+/// Display: Math Operator
+/// Category: math
+/// Returns: content
+#[func]
+pub fn operator(

```

Comment:
Do we really want these individual functions? Isn't `class("operator", ..)` fine?